applications/crossbar/src/modules/cb_notifications.erl

```diff
@@ -221,7 +221,7 @@ validate(Context) ->
     validate_notifications(maybe_update_db(Context), ReqVerb).
 
 validate(Context, ?SMTP_LOG) ->
-    load_smtp_log(Context);
+    crossbar_view:load(Context, ?CB_LIST_SMTP_LOG, fun(JObj) -> kz_json:get_value(<<"value">>, JObj) end);
```

Comment:
`fun(JObj) -> kz_json:get_value(<<"value">>, JObj) end`why not use something like `normalize_view_results ` ?